### PR TITLE
emergency meetings now dont call in mobs with no client

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

what the title says

## Why It's Good For The Game

this is unbearable on ice box or kilo

## Changelog
:cl:
fix: emergency meetings now dont call in mobs with no client
/:cl:

### CODE REVIEW

**Comment by MrDoomBringer** [comment]
he's saying what we were all thinking

File: code/__HELPERS/priority_announce.dm [hunk 1]
```diff
@@ -59,14 +59,12 @@
 	announcement += "<br><span class='alert'>[user] has called an Emergency Meeting!</span><br><br>"
 
 	for(var/mob/mob_to_teleport in GLOB.mob_list) //gotta make sure the whole crew's here!
```
Comment (MrDoomBringer) [inline code/__HELPERS/priority_announce.dm:61]:
```suggestion
	for(var/mob/mob_to_teleport in GLOB.player_list)
```
Don't need the `mob_to_teleport.client` check with this

**Review by MrDoomBringer — CHANGES_REQUESTED** [review]
You could also go ahead and add an `isaicamera(user)` check to the `can_make_emergency_meeting` proc if you wanted, I know some people complained about that

**Comment by TemporalOroboros** [comment]
Can this be used to antag check for nukies, wizards, abductors, and xenos?

**Comment by MrDoomBringer** [comment]
@TemporalOroboros no thats part of the fun

**Comment by Mothblocks** [comment]
nuke ops and wizards are an emergency to the station and so an emergency meeting for them makes sense

**Pull request merged**